_data/sw_servers.yml

```diff
@@ -219,6 +220,7 @@
           cap-3.1:
           cap-3.2:
           cap-notify:
+          channel-context-client-tag:
```

Comment:
I don't think it makes sense to mention it, except for Unreal it's redundant with message-tags. (and Unreal's also does not mention typing/react/reply)